schedulerTools.sh

```diff
@@ -127,4 +127,48 @@ function launch_job {
   rm ${sacct_log}
 }
 
-
+# Function to launch a job with the scheduler, or just run it if the scheduler is nonw
+function run_script {
+    local RUN_CMD=$1
+    local NAME=$2
+    local SCHEDULER_SCRIPT=$3
```

Comment:
Remove? Is this an argument or (as below) simply set using ${host} and ${scheduler}?

Reply:
I was thinking someone might optionally want to pass in a custom slurm script if the default the host was insufficient, so this would be an optional argument. Before, I was doing some sedding in that and passing in the script, but then realized I didn't need to do that for my use case in gt4py tests. But I could imagine wanting to add other things to the scheduler script, especially if you want to sed other values for the variables NTASKS, CPUSPERTASK etc, you could do it ahead of time, and pass that script in. If you think this makes it overly complicated I can remove it

Reply:
Oh I see I undid the logic that used the input arg file, good catch!